ui/src/app/pages/analysis/notebook-redirect.tsx

```diff
@@ -325,11 +323,13 @@ export const NotebookRedirect = fp.flow(withUserProfile(), withCurrentWorkspace(
             this.incrementProgress(Progress.Initializing);
           }
 
-          this.clearAndSetTimeout(() => this.pollForRunningCluster(billingProjectId), clusterPollingIntervalMillis);
+          this.pollTimer = setTimeout(() => this.pollForRunningCluster(billingProjectId), clusterPollingTimeoutMillis);
         }
-      } catch (e) {
-        if (!isAbortError(e)) {
-          throw e;
+      } catch (error) {
+        if (!isAbortError(error)) {
+          console.error(error);
```

Comment:
opt: maybe update this to use reportError instead, would be nice to have this logged as a UI error as well I think, since we don't have visibility into all of the API calls server side: https://github.com/all-of-us/workbench/blob/9473ca282c63015fe8d8c7f97258a3302aded80f/ui/src/app/utils/errors.tsx#L16